apertium-kaa.kaa.lexc

```diff
@@ -9734,6 +9736,7 @@ baxıt:baxıt N1 ;
 baxıtlı:baxıtlı A2 ;
 baxıtlılıq:baxıtlılıq N1 ;
 baxıtsız:baxıtsız A1 ;
+baxtıyar:baxtıyar A1 ; ! 
```

Comment:
Is this an adjective?  Could you give an example?

Reply:
Men baxtıyarman. I am happy.

Reply:
Okay, thanks, then this is good.